docs/k0s-single-node.md

```diff
@@ -1,53 +1,27 @@
 # Creating a single-node cluster
 These instructions outline a quick method for running a local k0s master and worker in a single node.
 
-> **_NOTE:_**  This method of running k0s is only recommended for dev, test or POC environments.
+ **_NOTE:_**  This method of running k0s is only recommended for dev, test or POC environments.
+ 
+## Prerequisites
 
-## Prepare dependencies
-#### 1. Download the k0s binary
-```sh
-curl -sSLf https://get.k0s.sh | sh
-```
-
-#### 2. Download the kubectl binary
-```sh
-sudo curl --output /usr/local/sbin/kubectl -L "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
-```
-
-#### 3. Make both binaries executable
-```sh
-sudo chmod +x /usr/local/sbin/kubectl
-sudo chmod +x /usr/bin/k0s
-```
+Install k0s as documented in the [installation instructions](k0s-install.md).
 
 ## Start k0s
-#### 1. Create the k0s config directory
+#### 1. Generate a default cluster configuration
 ```sh
-mkdir -p ${HOME}/.k0s
+k0s default-config > k0s.yaml
 ```
 
-#### 2. Generate a default cluster configuration
+#### 2. Start k0s
 ```sh
-k0s default-config | tee ${HOME}/.k0s/k0s.yaml
+$ sudo k0s install controller -c k0s.yaml --enable-worker
```

Comment:
question: would the systemd service be able to resolve the full path of k0s.yaml file from this command?

Reply:
No it won't 
```
[Service]
StartLimitInterval=5
StartLimitBurst=10
ExecStart=/root/go/src/github.com/k0sproject/k0s/k0s controller --config=val.yaml
```

good catch